src/components/Controls.vue

```diff
@@ -358,6 +365,13 @@ export default {
 				this.setPageTitle(current.title)
 			}
 		},
+		stacksByBoard: {
```

Comment:
I think this watch is not necessary.

Reply:
Could you please explain why this watcher is necessary even though the `cardsCount` is a computed variable?

Reply:
@miqmago